tests/http/test_server.py

```diff
@@ -308,3 +311,48 @@ def test_can_wrap_wsgi_apps(self):
         self.assertEqual(200, response.code)
         self.assertIn(
             'Hello, world!', tornado.escape.to_unicode(response.body))
+
+
+def default_webapp_factory(config, core):
+    class MainHandler(tornado.web.RequestHandler):
+        def get(self):
+            self.write("Hello from default webapp")
+
+    return [('/', MainHandler, {})]
+
+
+class HttpServerWithAppDefaultWebClient(tornado.testing.AsyncHTTPTestCase):
```

Comment:
We need a test in the case where the config value was invalid/unknown.

Reply:
Hello !
It is done, in commit 
https://github.com/mopidy/mopidy/pull/1791/commits/3ef06c11d00baa7102631fb6d949eccfd8cd10d2
Best